README.md

```diff
@@ -1,16 +1,341 @@
-# Title
+# Object Orientation in JavaScript
+
+“Every application is a collection of code; the code’s arrangement is the
+design." - Sandy Metz
 
 ## Learning Goals
 
--SWBAT 1
--SWBAT 2
+- Review the meaning of Object Orientation
+- Review the benefits of Object Orientation
+- Introduce how JavaScript can be written using Object Orientation
 
 ## Introduction
 
-## SWBAT 1
+So far in JavaScript, we've discussed the "Three Pillars of Web Programming",
+**Recognizing JS events**, **Manipulating the DOM**, and **Communicate with the
+server**, and we saw how these pillars are incorporated into web applications.
+
+The process we've learned has been straightforward:
+
+1.  An HTML page renders on the screen
+2.  JavaScript is executed when the page loads
+3.  In the JavaScript, event listeners are created pointing to 'free-standing'
+    functions, listed out in the JavaScript file.
+
+- Some of these functions handle DOM manipulation
+- Some functions handle server communication
+- Still other functions may serve as 'helper' functions to others
+
+This set up works great when just getting off the ground; with a handful of
+functions, we can get an interactive website up and running. What happens,
+though, when we want to go beyond a small website?
+
+Let's say we want to add feature, so we write a few more functions. Another
+function to handle a new event. That function needs to call a separate function
+for some DOM manipulation, then maybe a third function for updating our server.
+Meanwhile, it turns out, we can write a helper function that not only helps the
+code we just wrote, but cleans up some previous code we've written.
+
+Then we decide to add another, and more functions are needed. As we use our
+website, we think of more and more things to add or adjust. Pretty soon, the
+code has grown quite long. What's worse, all of the different things that happen
+on our website are starting to get jumbled together in a heap of functions. You
+may be able to get everything working for every feature we add, but coding this
+way comes at a cost:
+
```

Comment:
This all works, but something to consider might be getting more specific with the examples rather than leaving them generalized. We could refer back to our original example app, [Simple Liker](https://github.com/learn-co-curriculum/fewpjs-stitching-together-the-three-pillars), and describe which specific features we might want to add: number of people who have liked the post, linked list of people who have liked the post, etc.

Reply:
swapped out for simple liker narrative